client/src/app/(onboarding)/signup/page.tsx

```diff
@@ -1,3 +1,171 @@
-export default function Page() {
-  return <h1>Signup placeholder</h1>;
+"use client";
+import { Button, TextField, Typography, Link, Divider } from "@mui/material";
+import "./signup.css";
+import { useState } from "react";
+/*
+This page will represent the signup page of the application,
+which include the signup with google button, email, confirm email, and password fields
+and logging in link.
+*/
+
+export default function SignupPage() {
+  const [email, setEmail] = useState("");
```

Comment:
this is fine for placeholder for now but we will use Next Server Actions which might change a lot on this code in the future once we got our backend running. More on Server Actions in this documentation.

https://nextjs.org/docs/app/building-your-application/data-fetching/server-actions-and-mutations

Reply:
I will temporary leave it for now. Will comeback and fix it in the future.

Reply:
all fixed